rfcs/0014-address-encoding.md

```diff
@@ -0,0 +1,58 @@
+## Summary
+[summary]: #summary
+
+Change address encoding from Base64 to Base58.
+
+## Motivation
+
+[motivation]: #motivation
+
+First, it's a standard format across other well known cryptocurrencies (Bitcoin, Zcash, Monero). That means most entities in the ecosystem, from end-users to exchanges, will be more familiar and comfortable with Base58 encodings than what we have currently.
+
+Additionally, quoting from the [Bitcoin Wiki](https://en.bitcoin.it/wiki/Base58Check_encoding#Background): 
+
+```
+// Why base-58 instead of standard base-64 encoding?
+// - Don't want 0OIl characters that look the same in some fonts and
+//      could be used to create visually identical looking account numbers.
+// - A string with non-alphanumeric characters is not as easily accepted as an account number.
+// - E-mail usually won't line-break if there's no punctuation to break at.
+// - Doubleclicking selects the whole number as one word if it's all alphanumeric.
+
+```
+
+## Detailed design
+
+[detailed-design]: #detailed-design
+
+I recommend we follow the Bitcoin design, as laid out [here](https://en.bitcoin.it/wiki/Base58Check_encoding#Base58_symbol_chart).
+
+We should also add a unique prefix to prevent Coda from being accidentally sent to Bitcoin addresses or vice versa.
```

Comment:
can you specify this prefix here that we'll use in the parity-check hashing step as @enolan proposed (just pick an arbitrary string -- "coda address" as he suggested is probably good)

Reply:
done